src/Raven.Studio/typescript/viewmodels/shell.ts

```diff
@@ -275,10 +276,17 @@ class shell extends viewModelBase {
         super.attached();
 
         if (this.clientCertificate() && this.clientCertificate().Name) {
+            const dbAccessArray = certificateModel.resolveDatabasesAccess(this.clientCertificate());
+
+            const allowedDatabasesText = dbAccessArray.length ?
+                dbAccessArray.map(x => `<div>
+                                            <strong>${x.dbName}</strong>
+                                            <span class="${accessManager.default.getAccessColor(x.accessLevel)} margin-left">
+                                                         ${accessManager.default.getAccessLevelText(x.accessLevel)}
+                                            </span>
+                                        </div>`).join("")
+                : "Access to all databases is denied";
```

Comment:
we didn't deny access - we simply didn't granted anything ;-)

Reply:
How about just:  `"No access granted"` ?

Reply:
yes I think it is better, as we grant access, not deny access.

Reply:
done